Xamarin.Essentials/AssemblyInfo/AssemblyInfo.shared.cs

```diff
@@ -5,3 +5,4 @@
 [assembly: InternalsVisibleTo("XamarinEssentialsDeviceTestsUWP")]
 [assembly: InternalsVisibleTo("XamarinEssentialsDeviceTestsShared")]
 [assembly: InternalsVisibleTo("XamarinEssentialsDeviceTestsiOS")]
+[assembly: InternalsVisibleTo("Xamarin.Forms.Core")]
```

Comment:
Not sure about this... Do we need to? Can they just pull in their own from source?

Reply:
Yeah likely they will keep doing this for now.

Reply:
Removed.